RELEASENOTES.md

```diff
@@ -0,0 +1,3 @@
+### New in 5.0.0
```

Comment:
I think you might have misunderstood the release pipeline proposal.

1. You (only commits by haf will trigger it) want to release a new version.

2. You add at the _top_ of the RELEASENOTES.md file something like this:
``` markdown
### New in 6.0.0 (Released in 2017/12/14)

* Added support for .NET Core 3.5
```

3. You update the `.semver` file accordingly.

4. You make a commit that begins with the phrase `bumped version` (case-insensitive) and push it.

5. AppVeyor will see it, build the packages, push them to NuGet, make a Git tag, and push it too. It will not build the project _again_, after the tag is pushed.

The release notes are updated, _not_ replaced. I just did not add the previous release notes. It would be tedious.